src/DynamoCoreWpf/ViewModels/Watch3D/Shaders/Custom.fx

```diff
@@ -123,6 +119,12 @@ float4 PShaderCustom(PSInputCustom input) : SV_Target
 		I = I * input.c;
 	}
 
+    /// unselected geometry is transparent under Isolate Selected Geometry mode
+    if (isIsolationMode && !isSelected)
+    {
+        I.a = sqrt(I.a) * vTransparentMaterial.a;
+    }
+
```

Comment:
This is the direct negation of `(!isIsolationMode || isSelected)` condition, it is moved here because we need `I` to be assigned first, I guess. Do we not need to move `I.a = vMaterialDiffuse.a;` also in that case?